packages/common/src/types/user.ts

```diff
@@ -7,16 +7,26 @@ export interface LightdashUser {
     email: string | undefined;
     firstName: string;
     lastName: string;
-    organizationUuid: string;
-    organizationName: string;
-    organizationCreatedAt: Date;
+    organizationUuid?: string;
+    organizationName?: string;
+    organizationCreatedAt?: Date;
     isTrackingAnonymized: boolean;
     isMarketingOptedIn: boolean;
     isSetupComplete: boolean;
-    role: OrganizationMemberRole;
+    role?: OrganizationMemberRole;
     isActive: boolean;
 }
 
+export type LightdashUserWithOrg = Required<LightdashUser>;
+
+export const isUserWithOrg = (
+    user: LightdashUser,
+): user is LightdashUserWithOrg =>
+    user.organizationUuid !== undefined &&
```

Comment:
I'd say if ANY of the fields is undefined, then the user has no org: by replacing && with ||

Reply:
If I change to `or` logic I would have to wrap it with a negative since I want to return true if the user has an org. 
```!(user.organisationUuid !== undefined || user.role !== undefined)``` 
Not as readable imo.